docs/clients/dotnet/21.2/embedded/README.md

```diff
@@ -0,0 +1,192 @@
+# Embedded client
```

Comment:
See previous comment about embedded client

Reply:
Nice catch - removed in https://github.com/EventStore/documentation/pull/288/commits/ba3f7e7c27574e7c876a5ae749868472ce656f95.